content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,6 +7,17 @@ menu:
     name: Release notes
 ---
 
+## v1.6.4 [2022-03-15]
+
+### Features
+
+- Add `SASL` support to `Kafka` alerts.
+
+### Bug fixes
+
+- Fully deprecate DES based ciphers, RC4 based ciphers and TLS 1.1 ciphers. 
+- Update `Kafka` client to use the `Sarama` Go client library.
```

Comment:
line 19 isn't really a bug fix.
line 18 should be not just TLS 1.1 but anything TLS1.1 or older.

Reply:
Go ahead and remove it.  It's an internal implementation detail and not really customer facing.